static/app/views/dashboards/widgetBuilder/components/datasetSelector.tsx

```diff
@@ -81,6 +81,6 @@ const FeatureBadgeAlignmentWrapper = styled('div')`
 
 const Header = styled('h6')`
   font-size: ${p => p.theme.fontSizeLarge};
+  ${p => p.theme.tooltipUnderline()};
```

Comment:
Oh wait, I remembered, there's a `showUnderline` prop you can pass to `Tooltip`, does that remove this styling?

Reply:
ahh good catch I didn't know that was a thing! and it does the same styling 👍